### ISSUE TITLE
How can i download YOLO V8 🚀, GPL-3.0 license?

### ISSUE THREAD

**Issue by Aflexg** [ultralytics/ultralytics#8273]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello.I have programm, based on  YOLO v8, GPL-3.0 license.I want to use this programm with a new weight, which was trained on  YOLO v8, AGPL-3.0 license.Unfortunatelly it doesnt work.So i want to train it one more time on the same yolov8 version to sovle this problem.
Where and how can i get YOLO v8, GPL-3.0 license?

### Additional

Also i have the same problem/questiion for the yolov5.Where and how can i get YOLO v5, GPL-3.0 license?

**Comment by glenn-jocher** [comment]
@Aflexg hello! 👋

To download YOLOv8 under the GPL-3.0 license, you can clone the repository directly from GitHub using the following command:

```bash
git clone https://github.com/ultralytics/ultralytics.git
```

Once cloned, you can train your model with the desired weights. If you encounter any issues during training or have further questions, feel free to reach out!

Regarding YOLOv5, the process is similar. You can clone the YOLOv5 repository with:

```bash
git clone https://github.com/ultralytics/yolov5.git
```

Make sure to check out the respective README files in the repositories for additional guidance on training and using the models. Happy coding! 😊

**Comment by Aflexg** [comment]
Thanks for your reply.But, unfortunatelly it doesnt work.I created new folder and did the command git clone https://github.com/ultralytics/ultralytics.git. But it seems, that the license of the Yolov8 is still AGPL and i need a specific instruction to get GPL license.
![image](https://github.com/ultralytics/ultralytics/assets/153363231/3259dd21-7ced-4d59-a0f6-047f61f36bed)

**Comment by Aflexg** [comment]
Hello.Thanks for the reply.Unfortunatelly it doesnt work.I created yolov8 folder on the d disk and did this command : git clone https://github.com/ultralytics/ultralytics.git.As a result i have:
![image](https://github.com/ultralytics/ultralytics/assets/153363231/1da09e4f-c73c-4600-b8e4-f8960b3ca1df)
So, it seems, i still have AGPL version and need specific instruction for the instalaltion of the gpl version.

**Comment by Aflexg** [comment]
Where can i get the yolov8 with this version of the predict.py ?
[predict.txt](https://github.com/ultralytics/ultralytics/files/14329500/predict.txt)

**Comment by glenn-jocher** [comment]
@Aflexg oh sorry for the confusion, Ultralytics YOLOv8 is licensed under AGPL-3.0. There are no current GPL versions available, only AGPL-3.0.

**Comment by Aflexg** [comment]
That is so sad to hear.
